src/Plot.tsx

```diff
@@ -130,6 +134,29 @@ export default function Plot(props: PlotProps) {
               </g>
 
               {axes}
+
+              <Tracking
+                onClick={(position) => {
```

Comment:
Could pass undefined if callback is undefined

Reply:
More importantly, I think we should not render the <Tracking component at all if there are no callbacks, to fully optimize the common case.